Makefile

```diff
@@ -8,9 +8,10 @@ help:
 	@echo "  cli           run redis-cli inside the container on the server with port 7000"
 
 build:
-	docker-compose build
+	docker buildx build --build-arg redis_version=6.2.1 --platform linux/amd64,linux/arm64 --tag grokzen/redis-cluster:latest .
```

Comment:
Another major issue with forcing building both here is that most people don't care about having both, they only need one or the other. It might be more logical to make two different make targets instead of forcing everyone to build two different images. keep amd64 for `make build` and maybe make `make build-arm` or something that does the other version.

Reply:
Good point! `buildx` works by producing a single docker manifest with entries for each supported platform type. When consuming the image from a remote registry (e.g. dockerhub), the client can easily figure out which layers it needs for it's native platform and only download those via the manifest. So there's no additional cost for users who consume the image via a registry.

Unfortunately, `buildx` only supports building all platform types at once when you want to combine them into one manifest. To help streamline the experience of users who consume this image by building it locally, I could add a `make build-native` that builds a manifest that only supports the local CPU architecture. This way you would have

- `make build` => This is the release build that needs to be published to dockerhub. Supports AMD64 and ARM64.
- `make build-native` => This is a build that will only work on the native CPU architecture and shouldn't be published. 

See https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/

Let me know what you think! I'm happy to just add the `make build-native` command.

Reply:
No i am more inclined to go the other way around. Keep build as it was to keep backwards compatibility and then make a special `make buildx` or something that will work with these new features. It is much more pleasant to let the users opt-in during a trtransition phase and meek the old way it was until docker continues to build and evolve these features. I do not want to force a user into using a minimum docker version that is way to new and that might not even have install package in some distros. Many people still use super old dockers and the old way still works and should work for a long time to come imo.